docker-compose-dev/docker-compose.yml

```diff
@@ -1,13 +1,24 @@
 services:
   mosquitto:
-    image: eclipse-mosquitto:latest
+    image: eclipse-mosquitto:2.0.18
```

Comment:
can we ignore this issue? it's better to keep it latest so no need to update in future.

Reply:
I agree.  I can change it and mark it as intentional.